src/App.js

```diff
@@ -1,24 +1,27 @@
 import React from 'react';
 import './App.css';
+import GoodsList from './GoodList';
+import { goodsFromServer } from './goodsFromServer';
 
-const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
+class App extends React.Component {
+  state = {
+    isVisible: true,
+  }
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+  start = () => {
+    this.setState({ isVisible: false });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1>Goods</h1>
+        {(this.state.isVisible)
+          ? <button type="button" onClick={this.start}>Push the button</button>
+          : <GoodsList goods={goodsFromServer} />}
```

Comment:
you don't need it anymore in the props since you have an import in GoodsList file
`<GoodsList />`